### PR TITLE
add SKALE chain: Europa

### PR DESCRIPTION
**Pull request by RubyAquaMarine**

### CODE REVIEW

File: _data/chains/eip155-2046399126.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      "decimals": 18
+    },
+    "infoURL": "https://europahub.network/",
+    "shortName": "europa", 
+    "chainId": 2046399126,
+    "networkId": 2046399126,
+    "explorers": [
+      {
+        "name": "Blockscout",
+        "url": "https://elated-tan-skat.explorer.mainnet.skalenodes.com/",
```
Comment (ligi) [inline _data/chains/eip155-2046399126.json:22]:
```suggestion
        "url": "https://elated-tan-skat.explorer.mainnet.skalenodes.com",
```

File: _data/chains/eip155-2046399126.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        "standard": "EIP3091"
+      }
+    ],
+    "parent": {
+      "type": "L2",
+      "chain": "eip155-1",
+      "bridges": [{ "url": "https://ruby.exchange/bridge.html" }]
+    }
+  }
+
```
Comment (ligi) [inline _data/chains/eip155-2046399126.json:32]:
```suggestion
```

File: _data/icons/europa.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+      "url": "ipfs://QmPKW7WnAie3rMkjdqfF1cgK8UMZ8RgoUebhNaMZPBFgYp",
+      "width": 512,
+      "height": 512,
+      "format": "png"
+    }
+  ]
+
```
Comment (ligi) [inline _data/icons/europa.json:9]:
```suggestion
```

**Review by ligi — CHANGES_REQUESTED** [review]
where did you pin the IPFS CID? I could not resolve it via `ipfs get`

**Comment by RubyAquaMarine** [comment]
> where did you pin the IPFS CID? I could not resolve it via `ipfs get`

https://gateway.pinata.cloud/ipfs/QmPKW7WnAie3rMkjdqfF1cgK8UMZ8RgoUebhNaMZPBFgYp

**Comment by ligi** [comment]
please pin it somewhere else - pinata is currently not publishing to the DHT
also run the prettier

**Comment by RubyAquaMarine** [comment]
everything ok? @ligi

**Comment by RubyAquaMarine** [comment]
I'd love to check this off my todo list. I hope you're doing well @ligi

**Review by RubyAquaMarine — COMMENTED** [review]
changes added

**Comment by RubyAquaMarine** [comment]
anything else required? @ligi

**Comment by RubyAquaMarine** [comment]
@ligi  prettier ran again. npx prettier --check '_data/*/*.json' passes locally.

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/europa.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://bafkreiezcwowhm6xjrkt44cmiu6ml36rhrxx3amcg3cfkcntv2vgcvgbre",
+    "width": 512,
+    "height": 512,
+    "format": "png"
```
[the remark this answers is not in the document]
Comment (RubyAquaMarine) [reply _data/icons/europa.json]: ok

**Review by ligi — APPROVED** [review]

**Pull request merged**